gendox-core-api/src/main/java/dev/ctrlspace/gendox/gendoxcoreapi/discord/ListenerService.java

```diff
@@ -201,4 +200,99 @@ public String getTheQuestion(SlashCommandInteractionEvent event) {
     }
 
 
-}
+    public List<DocumentInstanceSection> findClosestSectionWithHttpRequest(String token, Message message, UUID projectId) throws GendoxException {
+        List<DocumentInstanceSection> sectionList = new ArrayList<>();
+        // Prepare the HTTP headers with the bearer token
+        HttpHeaders headers = new HttpHeaders();
+        headers.setBearerAuth(token);
+
+
+        // Prepare the request entity with the message and headers
+        HttpEntity<Message> requestEntity = new HttpEntity<>(message, headers);
+
+        // Create a RestTemplate
+
+        String url = "http://localhost:8080/gendox/api/v1/messages/semantic-search?projectId=" + projectId.toString() + "&page=0&size=5";
```

Comment:
This must come from application properties:
- gendox.domain.base-url
- server.servlet.context-path
- gendox.domain.messages.completion.post-completion-message
- gendox.domain.messages.completion.post-completion-message

Reply:
default page size should also be application property